images/plugin-compiler/data/build.sh

```diff
@@ -1,6 +1,7 @@
 #!/bin/bash
 set -xe
 
+CURRENTVERS=$(perl -n -e'/v(\d+).(\d+).(\d+)/'' && print "v$1\.$2\.$3"' ./../../../gateway/version.go)
```

Comment:
The [entry point](https://github.com/TykTechnologies/tyk/blob/TT-4354-feature/plugins-support-multiple-tyk-versions/images/plugin-compiler/Dockerfile#L22) for plugin compiler is this `build.sh` at path `/build.sh`. So `./../../../gateway/version.go` might not resolve correctly. gw code is copied to `TYK_GW_PATH` [here](https://github.com/TykTechnologies/tyk/blob/TT-4354-feature/plugins-support-multiple-tyk-versions/images/plugin-compiler/Dockerfile#L19). `./../../../gateway/version.go` can be changed to `$TYK_GW_PATH/gateway/version.go`